### PR TITLE
Remove tokio from dependencies

### PR DESCRIPTION
**Pull request by kostekIV**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,18 +8,16 @@ edition = "2018"
...
+log = "0.4"
+rand = "0.8"
+async-trait = "0.1"
 codec = { package = "parity-scale-codec", version = "2", default-features = false, features = ["derive"] }
-rand = "0.8.3"
-async-trait = "0"
 
 [dev-dependencies]
 libp2p = { version ="0", features = ["mdns", "tcp-tokio"]}
+tokio = { version = "0.2.21", features = ["macros", "rt-threaded", "time"] }
```
Comment (mike1729) [inline Cargo.toml:21]: Do we need macros and time now?

Comment (mike1729) [inline Cargo.toml:21]: And the most important thing, you can bump tokio version to the newest!

**Review by mike1729 — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (kostekIV) [reply Cargo.toml]: "time" was not needed but `macros` together with `rt` and `rt-multi-thread` are needed for tests because of `#[tokio::test]`.

Comment (kostekIV) [reply Cargo.toml]: Done

**Review by mike1729 — APPROVED** [review]

**Pull request merged**